### DISCUSSION TITLE
Typescript support with type2docfx now a dead project

### DISCUSSION THREAD

**Discussion by bartsipes** [dotnet/docfx#8602] [General]
I've been trying to follow the documentation on https://dotnet.github.io/docfx/tutorial/universalreference/gen_doc_for_ts.html for generating docs for a React website written in TypeScript. As I've worked my way through the issues, I've discovered that [type2docfx](https://github.com/docascode/type2docfx) hasn't been updated in nearly 3 years and the GitHub project was archived 2 years ago. Also, the [typedoc](https://typedoc.org/) package which is used to generate the yml files for type2docfx now appears to be able to generate a static page website itself now. 

Does anyone know know the state of support for TypeScript and if it even makes sense to try and generate docs using docfx? I'd like to for consistency with other docfx websites I have, but this really isn't seeming viable at this point.

**Comment by yufeih** [comment]
The open source version of `type2docfx` wasn't developed and maintained for a while. Universal reference for typescript and other languages aren't in a good supported state.

**Comment by danroot** [comment]
+1 for typescript support- or at least "here's where a contributor might go to help add it".  I did #8827 before I found this in discussions.  I'm documenting an ASP.NET Core + Angular/TS app and it would be nice to have guidance on the Angular side.
